src/components/Chat/PublishModalUserGroupFilter.tsx

```diff
@@ -0,0 +1,63 @@
+import { useCallback } from 'react';
+
+import { UserGoup } from '@/src/types/share';
+
+import { MultipleComboBox } from '../Common/MultipleComboBox';
+
+//TODO change to the real API call
+const userGroupsMock: UserGoup[] = [
+  { id: 'users-1', name: 'Users 1' },
+  { id: 'users-2', name: 'Users 2' },
+  { id: 'users-3', name: 'Users 3' },
+  { id: 'users-4', name: 'Users 4' },
+  { id: 'users-5', name: 'Users 5' },
+  { id: 'users-6', name: 'Users 6' },
+  { id: 'users-7', name: 'Users 7' },
+];
+
+//TODO remove from the file
+
+function getFilteredUserGroups<T>(
```

Comment:
it looks like it generic method for any dropdown items and should be renamed in generic way
```suggestion
function getFilteredItems<T>(
```

Reply:
Name changed and function moved to the util-file.